### DISCUSSION TITLE
LDAP integration help

### DISCUSSION THREAD

**Discussion by hmodateresyc** [turtle0x1/LxdMosaic#367] [Q&A]
Hi,

I configured the LDAP integration and have a few questions. (version 0.12.0)

1. the documentation says that there is a sync every hour and you can sync in the gui. Where in the gui can I force a sync? (the save button?)

2. Should the users be visible automatically in the users page or do I still need to create them locally?

3. I am not seeing any users. How can I troubleshoot this?

Thanks!

**Comment by turtle0x1** [comment]
are you using a container or snap?

**Reply by hmodateresyc** [reply]
container

**Reply by turtle0x1** [reply]
Your setting saved correctly? (I.E you clicked save and it didn't throw an error?)

If so can you run 

```
lxc exec lxdmosaic bash

cd /var/www/LxdMosaic 

vendor/bin/crunz schedule:list
```

**Reply by hmodateresyc** [reply]
It saved correctly (no errors).
It did give an error the first time but that was because php-ldap connect wasn't installed.
(I think it should be installed with the install script)

running crunz gives: No task found!

**Reply by turtle0x1** [reply]
Yes it should be installed by default I will open an issue for that!

> running crunz gives: No task found!

Is very odd, can you run `ls src/cronJobs` and check the files are found?

Can you also check `crunz.yml` you may have to set the `timezone` to the correct value

**Reply by hmodateresyc** [reply]
syncLdapUsersTask.php is there.

I changed the timezone.

settings that are saved:

{"userId": 1, "settings": [{"id": "4", "value": "ldap://10.0.0.5:389"}, {"id": "5", "value": "cn=Administrator,cn=Users,dc=lab,dc=be"}, {"id": "6", "value": "xxxxxxx"}, {"id": "7", "value": "ou=Users,dc=lab,dc=be"}]}

Still no users.

**Reply by turtle0x1** [reply]
this is strange, now you have updated the timezone does `vendor/bin/crunz schedule:list` still return no tasks? How about `vendor/bin/crunz schedule:list src/cronJobs/` ?

**Reply by hmodateresyc** [reply]
Still no tasks :(

**Reply by turtle0x1** [reply]
really strange, cant duplicate on my side which is a pain! I assume your up to date?

Any chance I can ssh in? You can get my email from `git log` and we can organise that way if possible!

**Reply by hmodateresyc** [reply]
It's a new install on a fresh ubuntu lxc. I'll send you an email.
